subtrack.MAUI/Shared/MainLayout.razor

```diff
@@ -2,7 +2,8 @@
 
 <div class="page">
     <main>
-        <article class="content px-4">
+        <!-- use padding-bottom of 5rem (equivalent to 80px) otherwise the navbar covers the bottom of the page content-->
```

Comment:
good fix, might be a good idea to reduce the navbar height aswell, on the images it looks like its got quite a lot of padding